api/services/model_load_balancing_service.py

```diff
@@ -346,11 +346,6 @@ def update_load_balancing_configs(
 
                 load_balancing_config = current_load_balancing_configs_dict[config_id]
 
-                # check duplicate name
-                for current_load_balancing_config in current_load_balancing_configs:
-                    if current_load_balancing_config.id != config_id and current_load_balancing_config.name == name:
```

Comment:
This line means that if the `config_id` is inconsistent during an update but matches the name of another config, an error is raised, which is expected behavior. 

I believe the root cause of this error should be checked. It might be that the update config ID is different from the previous one, or there could be other reasons.